### ISSUE TITLE
Logging DbCommands: All parameters are '?'

### ISSUE THREAD

**Issue by Wain123** [dotnet/efcore#8933]
I have an ILogger with the following function:
```
public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter) {
  Console.WriteLine(formatter(state, exception));
}
```
DbCommands get logged and the CommandText looks correct, but all the parameter values are simply '?'. Here's a sample output from the Logger:
```
Executed DbCommand (78ms) [Parameters=[p0='?', p1='?' (Size = 1), p2='?', p3='?' (Size = 6), p4='?', p5='?' (Size = 1), p6='?', p7='?', p8='?' (Size = 6)], CommandType='Text', CommandTimeout='0']
INSERT INTO FRM_LAYOUT_FILTER (SEQNR, BEZEICHNUNG, CREATETIME, CREATEUSER, DEFAULT_FILTER, FORM_NAME, MODIFY_NR, MODIFYTIME, MODIFYUSER)
VALUES (:p0, :p1, :p2, :p3, :p4, :p5, :p6, :p7, :p8)
```

**Comment by smitpatel** [comment]
By default EF doesn't put data in logs as the data may contain sensitive information. To get those parameter values, you need to chain `EnableSensitiveDataLogging()` after your `UseSqlServer` (or equivalent method)
e.g.
`builder.UseSqlServer("connectionstring").EnableSensitiveDataLogging()`

**Comment by jayoungers** [comment]
How is that not mentioned here?

https://docs.microsoft.com/en-us/ef/core/miscellaneous/logging

**Comment by smitpatel** [comment]
@divega For doc issue

**Comment by ajcvickers** [comment]
@divega Can this be moved out of the 2.1 milestone. (We are trying to make 2.1 only include issues that we must fix for 2.1)

**Comment by divega** [comment]
This issue was moved to aspnet/EntityFramework.Docs#673
